net/iputils/Makefile

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (C) 2006-2010 OpenWrt.org
+# Copyright (C) 2019 Noah Meyerhans <frodo@morgul.net>
+#
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iputils
```

Comment:
Can you reorder these to something like https://github.com/openwrt/packages/blob/master/libs/flac/Makefile#L10 ?

Makes it more consistent and easier to read.

Reply:
Done. And rebased against master.

Any last comments? Thanks for the review.